src/test/java/org/opentripplanner/graph_builder/module/DirectTransferGeneratorTest.java

```diff
@@ -250,22 +253,55 @@ public void testMultipleRequestsWithPatterns() {
     );
   }
 
+  @Test
+  public void testTransferOnIsolatedStations() {
+    OTPFeature.enableFeatures(Map.of(OTPFeature.NoTransfersOnIsolatedStops, true));
```

Comment:
This will change this feature for all tests run after this. If you keep the feature, you should use the `OTPFeature.NoTransfersOnIsolatedStops.testOn(() -> { /* test goes here */ })`

Reply:
This no longer a feature switch. I changed the test